pulp-dev.py

```diff
@@ -362,7 +362,7 @@ def install(opts):
         os.system('chmod 3775 /var/cache/pulp')
```

Comment:
Pretty sure this whole file is unused in Pulp 3. Want to remove in a separate commit on the same PR?